packages/react/src/theme/theme.tsx

```diff
@@ -37,7 +37,7 @@ function isThemeStyleSheet(styleSheet, theme: Theme) {
 }
 
 function switchThemeStyleSheets(enabledTheme: Theme) {
-  const disabledTheme = enabledTheme === 'dark' ? 'light' : 'dark';
+  const disabledTheme = themeNames[enabledTheme === themeNames[0] ? 1 : 0];
```

Comment:
Looks a bit weird with the array. Maybe it's better to make the themeNames the object and access the theme names by its keys?

Reply:
fix